device_calendar/ios/Classes/SwiftDeviceCalendarPlugin.swift

```diff
@@ -688,5 +753,31 @@ extension UIColor {
             return nil
         }
     }
+    
+    public convenience init?(hex: String) {
+        let r, g, b, a: CGFloat
+
+        if hex.hasPrefix("0x") {
+            let start = hex.index(hex.startIndex, offsetBy: 2)
+            let hexColor = String(hex[start...])
+
+            if hexColor.count == 8 {
+                let scanner = Scanner(string: hexColor)
+                var hexNumber: UInt64 = 0
+
+                if scanner.scanHexInt64(&hexNumber) {
+                    a = CGFloat((hexNumber & 0xff000000) >> 24) / 255
+                    r = CGFloat((hexNumber & 0x00ff0000) >> 16) / 255
+                    g = CGFloat((hexNumber & 0x0000ff00) >> 8) / 255
+                    b = CGFloat((hexNumber & 0x000000ff)) / 255
+                    
+                    self.init(red: r, green: g, blue: b, alpha: a)
+                    return
```

Comment:
should this return a value?  a couple of lines below it returns nil, so seems weird that this return doesn't actually return anything

Reply:
Yeah tested that it returns a wrong colour if I remove that `return` and build fails if I remove `return nil` on the few line below